project/DependsOn.scala

```diff
@@ -0,0 +1,21 @@
+import sbt._
+
+object DependsOn {
+
+  def scalatest(scalaVersion: String) =
+    if (scalaVersion.startsWith("2.12")) "org.scalatest" %% "scalatest" % "3.0.1" % Test
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal "2.12" appears 3 times in the file.](https://www.codacy.com/app/pathikrit/better-files/file/4972466329/issues/source?bid=2860326&fileBranchId=3676352#l6)